src/dbos-runtime/runtime.ts

```diff
@@ -39,7 +39,7 @@ export class DBOSRuntime {
       }
       await this.dbosExec.init(...classes);    
       const server = new DBOSHttpServer(this.dbosExec)
-      this.servers = server.listen(this.runtimeConfig.port);
+      this.servers = await server.listen(this.runtimeConfig.port);
```

Comment:
Nice catch!

Reply:
If the port is already in use, will this `listen` throw `EADDRINUSE`  ?

Reply:
> If the port is already in use, will this `listen` throw `EADDRINUSE` ?

Yes

Reply:
If we used it, would linter have complained about a dropped promise here?

Reply:
I thought the promise is handled in the listen method